macros/apply_masking_policies_for_model.sql

```diff
@@ -1,6 +1,9 @@
 {% macro apply_masking_policies_for_model() %}
     {% if execute %}
-        {% if config.get('materialized') in ['table', 'view'] %}
+        {% if config.get('materialized') == 'ephemeral' %}
+             {{ log("Skipping attaching masking policies for " ~ this.name ~ " ephemeral model", info=True) }}
+        {% elif config.get('materialized') in ['table', 'view'] or ('incremental' in config.get('materialized') and flags.FULL_REFRESH)
```

Comment:
Are we taking custom incremental into consideration?

Reply:
Yes, it will work for all models with `incremental` in materialization name